### ISSUE TITLE
Would this work?

### ISSUE THREAD

**Issue by EnderBrine101** [RedstoneLamp/RedstoneLamp#35]
Most of this is dummy, but is this code right? If so, I can do this for every block so when you get round to actually impementing them most of it will be done.

package redstonelamp.block;

public int getName(){
return "Air"
}

public int getId(){
return 0;
}

public int isBreakable(){
return false;
}

public int canBePlaced(){
return false;
}

public int isSolid(){
return false;
}

public int hasGravity(){
return false;
}

**Comment by PEMapModder** [comment]
This is wrong. Learn Java.

**Comment by EnderBrine101** [comment]
I read stack overflow.

**Comment by EnderBrine101** [comment]
it said use public int

**Comment by PEMapModder** [comment]
Use an IDE and it will tell you they are invalid types. You don't even know what int means. Please, this is not a place to teach you Java.

**Comment by EnderBrine101** [comment]
@PEMapModder Apprently its not public int  public void or public class.. it cant be public enum or public interface. http://ideone.com/xhEsPH

**Comment by EnderBrine101** [comment]
Int means integer I believe. Oh, yeah I see now.

**Comment by EnderBrine101** [comment]
I think I've got it. I was studying on the internet to find:

public String getName(){
return "Air";
}

**Comment by EnderBrine101** [comment]
Basically I need to do the <b>Below</b> In java:
https://github.com/PocketMine/PocketMine-MP/blob/master/src/pocketmine/block/Air.php

**Comment by PEMapModder** [comment]
RedstoneLamp isn't a port of PocketMine-MP or Bukkit.

**Comment by EnderBrine101** [comment]
Yes, i'm not trying to copy there code - im just using it to guide me. Besides, pocketmine is php so I <b>Couldnt</b> copy it (I dont know php).
But if we made a block.java then included it in all the <code>(blockname).java</code> files then we could determine whether each block is solid, has gravity etc. So basically if I got the right code for one <code>(blockname).java</code> then I could copy it for all other ones, with some exceptions such as editing ids, name etc. This would take a long time and i am willing to do it. I just canmot find the correct <code>code</code>

**Comment by PEMapModder** [comment]
And we never said that we are going to have the same functions as PocketMine or Bukkit.

**Comment by EnderBrine101** [comment]
Will all blocks be coded into one script?

**Comment by EnderBrine101** [comment]
If you dont want people posting here, by the way, people should be more active on the <b>forum</b>
